corgidrp/l1_to_l2a.py

```diff
@@ -1,9 +1,10 @@
 # A file that holds the functions that transmogrify l1 data to l2a data
-from corgidrp.detector import get_relgains, slice_section, detector_areas, flag_cosmics, calc_sat_fwc
+from corgidrp.detector import get_relgains, slice_section, detector_areas, flag_cosmics, calc_sat_fwc, imaging_slice, imaging_area_geom
 import numpy as np
 import corgidrp.data as data
 
-def prescan_biassub(input_dataset, noise_maps=None, return_full_frame=False, detector_regions=None):
+def prescan_biassub(input_dataset, noise_maps=None, return_full_frame=False, 
+                    detector_regions=None, use_imaging_area = False):
```

Comment:
Instead of adding in `use_imaging_slice`, which would need to be tested in the unit test for `prescan_biassub`, you could always `return_full_frame` here and take a subframe of the result in the trap calibration function right after it calls `prescan_biassub`.

Reply:
After looking at everything else and coming back to this, `use_imaging_slice` is implicitly tested by the trap-pump test, and I suppose it's okay to have redundant mechanisms which don't break anything else.